server/scripts/sendEmailBlast.js

```diff
@@ -0,0 +1,132 @@
+            please confirm your interest by clicking on the link below. Clicking on the link will reconfirm your interest in the program to ensure you remain visible to eligible employers.
+            <br/><br/>
+            <button>
+                <a href="${`${process.env.CLIENT_URL}/confirm-interest?id=${uuid}`}" rel=”noopener” target=”_blank”>Confirm Interest</a>
+            </button>
+            <br/><br/>
+            If you no longer wish to participate or be considered for the Health Career Access Program, please email us with the subject line WITHDRAW to <b>HCAPInfoQuery@gov.bc.ca</b>
+            <br/><br/>
+            <button>
+                <a href="mailto:HCAPInfoQuery@gov.bc.ca?subject=WITHDRAW&body=Hello,%20Please%20withdraw%20the%20expression(s)%20of%20interest%20for%20the%20sender%20of%20this%20email.%0d%0aI%20am%20no%20longer%20interested%20in%20participating%20in%20the%20Health%20Career%20Access%20Program.">Withdraw from HCAP</a> 
+            </button>
+        </body>
+        
+        `,
+  };
+}
+
+const config = (token) => ({
+  headers: {
+    authorization: `Bearer ${token}`,
+    'Content-Type': 'application/json',
+  },
+});
+
+async function getEmailBlock(index, block) {
+  return dbClient.db.query(
+    `SELECT email_address , otp FROM confirm_interest LIMIT ${block} OFFSET ${index}`
+  );
+}
+async function countEmails() {
+  await dbClient.connect();
+  return dbClient.db.query('SELECT Count(*) FROM confirm_interest');
+}
+
+const sleep = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
+
+async function sendEmail(email, otp, index, conf) {
+  try {
+    await axios.post(`${CHES_HOST}/api/v1/email`, createPayload(email, otp), conf);
+  } catch (e) {
+    failedSends.push(index);
+    console.log(e.response.data.errors);
+  }
+}
+
+async function blastRecursive(start, end, max, batch, chesConfirguration) {
+  if (start >= max || (start >= end && end !== -1)) {
+    return true;
+  }
+  console.log('Sending emails from index ', start, ' to ', start + batch);
+  const emails = await getEmailBlock(start, batch);
+  const promiseArr = emails.map((res, index) =>
+    sendEmail(res.email_address, res.otp, index + start, chesConfirguration)
+  );
+  await Promise.all(promiseArr);
+  await sleep(1000);
+  return blastRecursive(start + batch, end, max, batch, chesConfirguration);
+}
+async function blast(start, end, batch) {
+  const { count } = (await countEmails())[0];
+  console.log(count);
+  const token = await authenticateChes();
+
+  await blastRecursive(start, end, count, batch, config(token));
+}
+
+(async function emailBlast() {
+  const arglength = process.argv.length;
+  const mode = arglength > 2 ? process.argv[2] : 'default';
+  const start = arglength > 3 ? parseInt(process.argv[3], 10) : 0;
+  const end = arglength > 4 ? parseInt(process.argv[4], 10) : -1;
+  const batch = arglength > 5 ? parseInt(process.argv[5], 10) : 100;
```

Comment:
Should we add a little confirmation step here e.g. "Press enter to continue"? We could print out the settings that the blast will run with (start index, batch size, etc.). If one of these `parseInt`s fails, we'll have some unexpected result.